server/tests/conftest.py

```diff
@@ -41,3 +41,39 @@ def tram_trips_by_id() -> dict[str, list[int]]:
 def krakow_city_configuration():
     with open("tests/assets/krakow_city_configuration.json") as file:
         return CityConfiguration.model_validate_json(file.read())
+
+
+@pytest.fixture
+def krakow_ignored_crossings_and_stops():
+    with open("tests/assets/krakow_ignored_crossings_and_stops.json") as file:
+        return json.load(file)
+
+
+@pytest.fixture
+def osm_tram_track_crossings() -> overpy.Result:
+    with zipfile.ZipFile(
+        "tests/assets/tram_track_crossings_overpass_query_result.zip"
+    ) as zip_file:
+        with zip_file.open("osm_tram_track_crossings.pickle") as file:
+            osm_tram_track_crossings: overpy.Result = pickle.load(file)
```

Comment:
I think there's no need for type hints here, the one in the function signature should be enough:
```suggestion
            osm_tram_track_crossings = pickle.load(file)
```
(Applies to the entire file :3)